_data/simple-icons.json

```
@@ -12845,6 +12845,11 @@
                 "type": "CC-BY-NC-SA-4.0"
             }
         },
+        {
+            "title": "Rocket",
+            "hex": "D33847",
+            "source": "https://github.com/simple-icons/simple-icons/pull/9492#issuecomment-1726472160"
```

We need an actual source from where the logo was picked up. Not a comment from the PR itself. Well if the author or someone had provided in the PR comments, that would have been acceptable.

Anyway, how about we use the logo-small.svg from their own website itself?
```suggestion
            "source": "https://rocket.rs/"
```